### PR TITLE
Add e2e app test

### PR DESCRIPTION
**Pull request by iomodo**
#### Summary
This PR adds e2e test for the test-app
run `/make test-app-e2e`

### CODE REVIEW

File: server/config/service.go [hunk 1]
```diff
@@ -112,6 +112,9 @@ func (s *service) Reconfigure(stored StoredConfig, services ...Configurable) err
 }
 
 func (s *service) StoreConfig(sc StoredConfig) error {
+	if strings.HasPrefix(*s.mattermostConfig.ServiceSettings.SiteURL, "http://localhost:") {
```
Comment (iomodo) [inline server/config/service.go:115]: This is done because `s.mm.Configuration.SavePluginConfig` method is not working on an e2e environment.

**Comment by iomodo** [comment]
@levb @catalintomai kind reminder to review this.

**Comment by levb** [comment]
@catalintomai this seems relevant, but is behind, and should perhaps be done with the `test-e2e` framework? What shall we do with the PR?

**Comment by catalintomai** [comment]
@levb, I'd say let's close it. It seems fragile anyway and as you pointed out, we can come up with a better approach.

**Pull request closed without merging**